src/promptflow-rag/promptflow/rag/_build_mlindex.py

```diff
@@ -232,16 +281,29 @@ def _create_mlindex_from_existing_ai_search(
         mlindex_config["index"]["field_mapping"]["metadata"] = ai_search_config.ai_search_metadata_key
 
     model_connection_args: Dict[str, Optional[Union[str, Dict]]]
-    if not aoai_connection:
-        import openai
-
-        model_connection_args = {
-            "key": openai.api_key,
+    if "cohere" in embedding_model:
+        # api_key = "SERVERLESS_CONNECTION_KEY"
+        # api_base = "SERVERLESS_CONNECTION_ENDPOINT"
+        # connection_args = {
+        #     "connection_type": "environment",
+        #     "connection": {"key": api_key},
+        #     "endpoint": os.getenv(api_base),
+        # }
```

Comment:
Comment?

Reply:
this part of the code should be uncommented once azureml-rag support connecting to serverless endpoints with environment variables.